tests/api_data/dsm_7/photos/const_7_photo.py

```diff
@@ -38,6 +38,25 @@
                 "type": "normal",
                 "version": 195694,
             },
+             {
+                "cant_migrate_condition": {},
+                "condition": {},
+                "create_time": 1718658534,
+                "end_time": 1719075481,
+                "freeze_album": False,
+                "id": 3,
+                "item_count": 1,
+                "name": "Album3",
+                "owner_user_id": 7,
+                "passphrase": "NiXlv1i2N",
```

Comment:
This triggers a precommit error, perhaps we should just use dummy?

Reply:
Tried multiple values and still got the error. I disabled this rule for tests. Should be ok